### PR TITLE
Video and PDF Thumbnails

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issue**: 
- Part of https://github.com/Islandora-CLAW/CLAW/issues/1005
- Part of https://github.com/Islandora-CLAW/CLAW/issues/1111
- Depends on https://github.com/Islandora-CLAW/islandora/pull/130

# What does this Pull Request do?

Adds actions and context configuration to generate thumbnails for PDFs and Videos by extracting the first page/frame (respectively).

# What's new?

Some real basic config.  To prevent making unnecessary duplicates, I ended generating thumbnails from the original files instead of from service files.  Then that one action is used by both the Image and PDF contexts to generate thumbnails.

# How should this be tested?

https://github.com/Islandora-Devops/claw-playbook/pull/105

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: config/install/system.action.video_generate_a_thumbnail_from_an_original_file.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+label: 'Video - Generate a thumbnail from an original file'
+type: node
+plugin: generate_image_derivative
+configuration:
+  queue: islandora-connector-homarus
+  event: 'Generate Derivative'
+  source_term_uri: 'http://pcdm.org/use#OriginalFile'
+  derivative_term_uri: 'http://pcdm.org/use#ThumbnailImage'
+  mimetype: image/jpeg
+  args: '-ss 00:00:01.000 -frames 1'
```
Comment (seth-shaw-unlv) [inline config/install/system.action.video_generate_a_thumbnail_from_an_original_file.yml]: Can we add an arg to shrink the resulting "thumbnail" to a more thumbnail-like size? E.g. `scale=100:-2`?

Comment (seth-shaw-unlv) [reply config/install/system.action.video_generate_a_thumbnail_from_an_original_file.yml]: `-ss 00:00:01.000 -frames 1 -vf scale=100:-2`

Comment (dannylamb) [reply config/install/system.action.video_generate_a_thumbnail_from_an_original_file.yml]: Updated!  Awesome, thx @seth-shaw-unlv

**Comment by dannylamb** [comment]
@seth-shaw-unlv good to go on this now

**Comment by dannylamb** [comment]
I mean now it is ^_^

Didn't read your feedback on https://github.com/Islandora-Devops/claw-playbook/pull/105 until after i commented on all the pulls.

**Comment by dannylamb** [comment]
@seth-shaw-unlv Let me know if there's anything else needed for this

**Comment by seth-shaw-unlv** [comment]
@dannylamb I think we're good. I'm just going to spin this up one last time to double-check.

**Pull request merged**